packages/components/src/draggable/index.js

```diff
@@ -6,12 +6,11 @@ import { noop } from 'lodash';
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { Component, createRef } from '@wordpress/element';
```

Comment:
There's a lot of changes in this file, I just wanted to mention that this is a generic Draggable component which means it shouldn't be modeled with the block draggable in mind. If we want an option to remove the clonable, it shouldn't be automatic as it breaks backward compatibility.

Reply:
Hmm, indeed, that may affect this change I made: https://github.com/WordPress/gutenberg/pull/23024/commits/bac44fd5f6cf2caa9fb83f60f48da574a5e78dc1

I'm curious how to best proceed, as I'm getting in the deep end here. Do you have time to help me out with this one?

Reply:
I'm afraid I won't have time today but I can look tomorrow... ideally you should be able to use Storybook to work on this component  with multiple variations.

Reply:
Could we just revert to hiding the clone?

Reply:
What did we end up doing here?